bin/cml-pr.js

```diff
@@ -11,15 +11,17 @@ const CML = require('../src/cml');
 const run = async (opts) => {
   const globs = opts._.length ? opts._ : undefined;
   const cml = new CML(opts);
-  print(await cml.pr_create({ ...opts, globs }));
+  print((await cml.pr_create({ ...opts, globs })) || '');
 };
 
 const opts = decamelize(
   yargs
-    .strict()
-    .usage('Usage: $0 <path to markdown file>')
+    .usage('Usage: $0 <glob path> ... <glob path>')
     .describe('md', 'Output in markdown format [](url).')
     .boolean('md')
+    .default('git-remote', 'origin')
+    .default('git-user-email', 'olivaw@iterative.ai')
+    .default('git-user-name', 'iterative-olivaw')
```

Comment:
do we need the `git-` prefix? `--remote, --user-email, --user-name` are hopefully clear enough

```suggestion
    .default('remote', 'origin')
    .default('user-email', '64868532+iterative-olivaw@users.noreply.github.com')
    .default('user-name', 'Olivaw[bot]')
```

Reply:
I thought the same originally with the idea of explaining it in the describe or use that name to be explicit... still not sure whats better.

Reply:
I think there's no scenario where `--user-email`/`--user-name` could possibly mean anything else.

In fact, if in future they could mean something else we'd have to be explicit about the *other* meaning (`--github-handle`, `--notification-email`, etc)

Reply:
lets do a short version then, but I have to say that Im not totally sold here